.changeset/bubble-beach.md

```diff
@@ -0,0 +1,5 @@
+---
+"@asyncapi/multi-parser": patch
+---
+
+Adding the Way the generator is imported in Nunjucks filter.
```

Comment:
Improve the message so it reflects what it fixes.

Reply:
done.